.gitignore

```diff
@@ -1,7 +1,4 @@
 .gradle/
-gradle/
```

Comment:
Will these need to no longer be ignored for publishing reasons?

Reply:
They actually should not be ignored because we can't upgrade the Gradle version if they are. I had to upgrade Gradle to resolve a bug in the version we were using. That means we will need to be careful about changes to that directory but the only way that will happen is if someone explicitly upgrades Gradle or directly mucks around in it.

Reply:
Should we add upgrade instructions to the README?

Reply:
We can always just link to https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper